File: app/src/main/java/com/lcl/lclmeasurementtool/Constants/NetworkConstants.java
package com.lcl.lclmeasurementtool.Constants;

public class NetworkConstants {
    public static final String URL = "https://api-dev.seattlecommunitynetwork.org/api";
    public static final String REGISTRATION_ENDPOINT = "/register";
    public static final String SIGNAL_ENDPOINT = "/report_signal";
    public static final String CONNECTIVITY_ENDPOINT = "/report_measurement";

    // NetworkTestViewModel Constants
    public static final String PING_TAG = "PING";
    public static final String IPERF_UP_TAG = "IPERF_UP";
    public static final String IPERF_DOWN_TAG = "IPERF_DOWN";
    public static final String WORKER_TAG = "backgroundTest";
    public static final String IPERF_COUNTS_TAG = "TIMES";
    public static final int IPERF_COUNTS = 5;
    public static final String IPERF_TEST_ADDRESS = "google.com";
    public static final String IPERF_TEST_ADDRESS_TAG = "ADDRESS";
}

File: app/src/main/java/com/lcl/lclmeasurementtool/constants/NetworkConstants.kt
package com.lcl.lclmeasurementtool.constants

class NetworkConstants {

    companion object {
        const val URL = "https://coverage.seattlecommunitynetwork.org/api/"
        const val REGISTRATION_ENDPOINT = "register"
        const val SIGNAL_ENDPOINT = "report_signal"
        const val CONNECTIVITY_ENDPOINT = "report_measurement"
        const val MEDIA_TYPE = "application/json"

        // NetworkTestViewModel Constants
        const val PING_TAG = "PING"
        const val IPERF_UP_TAG = "IPERF_UP"
        const val IPERF_DOWN_TAG = "IPERF_DOWN"
        const val WORKER_TAG = "backgroundTest"
        const val IPERF_COUNTS_TAG = "TIMES"
        const val IPERF_COUNTS = 5
        const val PING_TEST_ADDRESS = "google.com"
        const val PING_TEST_ADDRESS_TAG = "ADDRESS"
    }

}